language/fr/login.lang

```diff
@@ -0,0 +1,28 @@
+<?php
+
+include("{$_EnginePath}language/en/login.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.